config/config.toml.example

```diff
@@ -3,6 +3,9 @@
 # TiDB server host.
 host = "0.0.0.0"
 
+# tidb server advertise IP.
```

Comment:
Will the address contain port? Or only IP?

Reply:
Only IP.

Reply:
Can we obtain the IP from system information?